opm/simulators/flow/BlackoilModelParametersEbos.hpp

```diff
@@ -141,7 +141,7 @@ struct RelaxedPressureTolMsw {
     using type = UndefinedProperty;
 };
 template<class TypeTag, class MyTypeTag>
-struct RegularizationFactorMsw {
+struct RegularizationFactor {
```

Comment:
Good to have a name not invoking msw, but perhaps RegularizationFactorWells would be better? Not sure if we would ever want to have a "general regularization factor".